test/export/test_export.py

```diff
@@ -2617,27 +2617,25 @@ def forward(self, x):
     @testing.expectedFailureSerDer  # no unbacked bindings after deserialization?
     def test_unbacked_bindings_for_divisible_u_symint(self):
         with torch.library._scoped_library("mylib", "FRAGMENT") as lib:
-            torch.library.define(
-                "mylib::foo",
-                "(Tensor a, Tensor b) -> (Tensor)",
+            lib.define(
```

Comment:
define should work in the way you implemented it. The problem is likely just that your register_fake call on like 2635 needs a lib=lib

Reply:
Ahh that makes sense